src/Repository/AdherentRepository.php

```diff
@@ -770,4 +771,22 @@ public function refresh(Adherent $adherent): void
     {
         $this->getEntityManager()->refresh($adherent);
     }
+
+    public function findPaginatedForInseeCodes(
+        string $inseeCodes,
```

Comment:
it should be an array here, with the string you cannot be sure that it follows a good format for `FIND_IN_SET` function (`val1,val2,valN`).